internal/exec/stages/files/files.go

```diff
@@ -123,6 +121,11 @@ func (s stage) runImpl(config types.Config, isApply bool, applyIgnoreUnsupported
 		if err := s.relabelFiles(); err != nil {
 			return fmt.Errorf("failed to handle relabeling: %v", err)
 		}
+
+		// !isApply: saves LUKS volume key
```

Comment:
I'd put this right after the `createCrypttabEntries()` since it's related.

Reply:
Will add next to `createCrypttabEntries()`

Reply:
Done